.eslintrc

```diff
@@ -0,0 +1,68 @@
+{
+    "extends": [
+        "eslint:recommended"
+    ],
+    "env": {
+        "node": true,
+        "es6": true
+    },
+    "parserOptions": {
+        "ecmaVersion": 2018,
+        "sourceType": "module"
+    },
+    "rules": {
+        "semi": "error",
+        "no-extra-semi": "error",
+        "newline-before-return": "error",
+        "no-var": "error",
+        "prefer-const": "error",
+        "object-shorthand": "error",
+        "prefer-arrow-callback": "error",
+        "prefer-numeric-literals": "error",
+        "prefer-template": "error",
+        "prefer-spread": "error",
+        "prefer-rest-params": "error",
+        "yoda": "error",
+        "callback-return": "error",
+        "handle-callback-err": "error",
+        "no-process-exit": "warn",
+        "no-buffer-constructor": "error",
+        "no-mixed-requires": "warn",
```

Comment:
Please don't use `warn` :)

Decide if it should be `right` or `wrong`.

Telling a developer that it is maybe wrong is not that helpful 😉

Reply:
Yeah, you're right but in some cases it maybe should't raise an error immediately (although I can't give you a quick example and if so you'd either harden the rule or allow it haha).
I'll mark all as error, never thought about this in that way so far :)

Reply:
Cool 👍 if it is not bad enough for an error then it is probably okay to allow it 😉